src/inc/unicode.hpp

```diff
@@ -26,37 +26,37 @@ Author(s):
 #define IsEastAsianCP(cp) ((cp) == CP_JAPANESE || (cp) == CP_KOREAN || (cp) == CP_CHINESE_TRADITIONAL || (cp) == CP_CHINESE_SIMPLIFIED)
 
 // UNICODE_NULL is a Windows macro definition
-const wchar_t UNICODE_BACKSPACE = 0x8;
-const wchar_t UNICODE_ESC = 0x1b;
-const wchar_t UNICODE_DEL = 0x7f;
+constexpr wchar_t UNICODE_BACKSPACE = 0x8;
+constexpr wchar_t UNICODE_ESC = 0x1b;
+constexpr wchar_t UNICODE_DEL = 0x7f;
 // NOTE: This isn't actually a backspace. It's a graphical block. But
 // I believe it's emitted by one of our ANSI/OEM --> Unicode conversions.
 // We should dig further into this in the future.

```

Comment:
This comment clearly belongs with `UNICODE_BACKSPACE2` rather than `UNICODE_BEL`. Would it be out of scope to fix that while you're making changes here? Also, why is there a `UNICODE_BEL` as well as a `UNICODE_BELL`? Do we really need both of them?

Reply:
Yeah `UNICODE_BEL` is not used in the codebase, AFAIK. So I removed it.